cmd/helm/template.go

```
@@ -126,6 +133,14 @@ func (t *templateCmd) run(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	// verify that output-dir exists if provided
+	if t.outputDir != "" {
+		_, err = os.Stat(t.outputDir)
+		if os.IsNotExist(err) {
+			panic(fmt.Sprintf("output-dir '%s' does not exist", t.outputDir))
```

This should return an error like the `fmt.Errorf()` call a few lines above.